src/test/java/nl/praegus/fitnesse/symbols/MavenProjectVersions/MavenProjectVersionsSymbolTest.java

```diff
@@ -0,0 +1,171 @@
+package nl.praegus.fitnesse.symbols.MavenProjectVersions;
+import org.junit.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class MavenProjectVersionsSymbolTest {
```

Comment:
Ik gok dat je in deze klasse veel logica test die in de VersionNumber klasse zit. Tests die specifiek die logica testen moeten naar een nieuwe testklasse, waar je specifiek VersionNumber instantieert en test. In deze testklasse moeten alleen tests zitten die specifiek MavenProjectVersionsSymbol testen.

Reply:
Unit testen toegevoegd